src/NextGen/dataHandler/dataStrategyManagerImplementation.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseEvent } from '@mparticle/event-models';
+import { IMParticleDataHandler } from './dataHandler';
+import { IMParticleDataHandlerStrategy } from './dataHandlerStrategy';
+
+// TODO: We should define some interface for the Input so that we can make sure
+//       we have the correct methods that we expect it to have
+export class MParticleDataStrategyManagerImplementation<Input, Output>
```

Comment:
I'm fine with dropping `Implementation` for web and just keeping `I` as shorthand for `Interface`. That's seems more standard for Web.

On the same note, I'm also in favor of dropping `Mparticle` from these classes and interfaces unless there's a real good reason to keep them. IIRC, the only reason we thought of keeping that prefix is because one of the Native platforms didn't support namespacing?